packages/slate-editor/src/components/OptionsGroup/OptionsGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+import type { OptionsGroupOption, RadioChoose } from './Option';
+import { ChooseOption } from './Option';
+import styles from './OptionsGroup.module.scss';
+
+interface OptionsGroupCommon<T extends string> {
+    name: string;
+    options: OptionsGroupOption<T>[];
+    columns?: number;
+}
+
+type OptionsGroupProps<T extends string> = OptionsGroupCommon<T> & RadioChoose<T>;
+
+export function OptionsGroup<T extends string>(props: OptionsGroupProps<T>) {
+    const allSelected = React.useMemo(() => new Set([props.selected]), [props.selected]);
```

Comment:
Because our `OptionsGroups` is always a radio group, the controlled state should only be one `value` prop, and not an array/set. Right?